android/src/main/java/com/oney/WebRTCModule/GetUserMediaImpl.java

```diff
@@ -109,7 +109,7 @@ private AudioTrack createAudioTrack(ReadableMap constraints) {
         AudioTrack track = pcFactory.createAudioTrack(id, audioSource);
         tracks.put(
             id,
-            new TrackPrivate(track, audioSource, /* videoCapturer */ null));
+            new TrackPrivate(track, audioSource, /* videoCapturer */ null, null));
```

Comment:
Could you please add a comment here to explain why it's fine to add null as surfaceTextureHelper

Reply:
done